infra/blueprint-test/pkg/krmt/krm.go

```diff
@@ -0,0 +1,312 @@
+package krmt
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+
+	gotest "testing"
+
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/discovery"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/git"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/kpt"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/utils"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/mitchellh/go-testing-interface"
+	"github.com/otiai10/copy"
+	"github.com/stretchr/testify/assert"
+)
+
+const tmpBuildDir = ".build"
+
+var CommonSetters = []string{"PROJECT_ID", "BILLING_ACCOUNT_ID", "ORG_ID"}
+
+// KRMBlueprintTest implements bpt.Blueprint and stores information associated with a KRM blueprint test.
+type KRMBlueprintTest struct {
```

Comment:
A lot of this could be combined with the Terraform version (ex. name, logger, etc.). Do you plan to consolidate in a later PR?

Reply:
Yup, along with the binary helpers too.